templates/docs/patterns/tab-buttons.md

```diff
@@ -0,0 +1,62 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Tab Buttons | Components
+---
+
+## Tab Buttons
```

Comment:
I think we could still keep this as part of existing "Tabs" documentation page, same as on "Navigation" we have different components that do navigation. Especially that `p-tabs` and `p-tab-buttons` are functionally very similar.

If we want to keep them on separate pages (I'm fine with that, it is a separate component) we should maybe link between each other. So "Tabs" page would mention and link to "Tab buttons" and vice versa.